From 0d1c4d505dd8037aeea2191fce0ff5a09f4a87f9 Mon Sep 17 00:00:00 2001
From: Mitch Luban <mluban@nvidia.com>
Date: Fri, 21 Dec 2012 11:34:20 -0800
Subject: [PATCH] power: allow easy access to number of powerhal hints

Add a POWER_HINT_COUNT to power hints.

Bug 1183244

Reviewed-on: http://git-master/r/173711
(cherry picked from commit 7034546f030ffdb3f588a861843cce030d8b8bdc)

Change-Id: Iab10da1e043737f9e0a2c837f985f6cb5e11811a
Reviewed-on: http://git-master/r/189656
Tested-by: Mitch Luban <mluban@nvidia.com>
Reviewed-by: Prajakta Gudadhe <pgudadhe@nvidia.com>
---
 include/hardware/power.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/hardware/power.h b/include/hardware/power.h
index 6c55061..ee266f3 100644
--- a/include/hardware/power.h
+++ b/include/hardware/power.h
@@ -41,6 +41,7 @@ __BEGIN_DECLS
 typedef enum {
     POWER_HINT_VSYNC = 0x00000001,
     POWER_HINT_INTERACTION = 0x00000002,
+    POWER_HINT_COUNT,
 } power_hint_t;
 
 /**
-- 
1.8.1.5

